locales/en/messages.json

```diff
@@ -3383,12 +3383,24 @@
     "powerBatteryWarning": {
         "message": "Warning Cell Voltage"
     },
+    "powerCalibrationManager": {
+        "message": "Calibration Manager"
+    },
+    "powerCalibrationManagerHelp": {
+        "message": "<strong>Note:</strong> Before calibrating the scale make sure that divider and multiplier for voltage and offset for amperage is set properly.<br>Setting the values to actual voltage or amperage will calibrate the respective meter scales. Leaving it at 0 will not apply calibration.</br>"
```

Comment:
I think the message is a bit confusing for a user who does not know what this is about.
I would remove 'Setting the values to actual voltage or amperage will calibrate the respective meter scales.' from the note, and instead put the instructions on top, above the note, like:

'To calibrate, use a multimeter to measure the actual voltage / current draw on your craft (with a battery plugged in), and enter the values below. Then, with the same battery still plugged in, click [Calibrate].'

Also, please add a 'remove props before plugging in a battery' to the note.

Reply:
I added the instruction, and prop warning.